README.md

```diff
@@ -99,6 +99,17 @@ reporter = file_reporter.FileReporter(file_name='traces')
 tracer = context_tracer.ContextTracer(reporter=reporter)
 ```
 
+Report to Stackdriver Trace:
```

Comment:
This clashes with your README.md -> README.rst, so doublecheck that these changes don't get lost in a merge conflict.

Reply:
Thanks for the reminder! Merged and doublechecked.